immersive-web-wg-charter.html

```diff
@@ -1,717 +1,971 @@
+        <p>The following features are out of scope, and will not be addressed by this Working group.</p>
+
+        <ul class="out-of-scope">
+          <li>Defining browser user experience inside virtual or augmented reality, aside from navigating between XR
+            sites.</li>
+          <li>Defining mechanisms for global-scale AR browsing.</li>
+        </ul>
+      </section>
 
+    </section>
 
-    <main>
-      <h1 id="title">[PROPOSED] Immersive Web Working Group Charter</h1>
-      <!-- delete PROPOSED after AC review completed -->
+    <section id="deliverables">
+      <h2>
+        Deliverables
+      </h2>
 
-      <p class="mission">
-        The <strong>mission</strong> of the <a href="https://www.w3.org/immersive-web/">Immersive Web Working Group</a> is to help bring high-performance Virtual Reality (VR) and Augmented Reality (AR) (collectively known as XR) to the open Web via APIs to interact with XR devices and sensors in browsers.
+      <p>More detailed milestones and updated publication schedules are available on the <a
+          href="https://www.w3.org/immersive-web/" target="_blank">group publication status page</a>.
       </p>
-      <!-- the link to the group is ALWAYS that available from https://www.w3.org/groups/wg or https://www.w3.org/groups/ig because each group page can then point to a different homepage, but not the other way around. -->
 
-      <div class="noprint">
-        <p class="join"><a href="https://www.w3.org/2004/01/pp-impl/109735/join">Join the Immersive Web Working Group.</a></p>
-      </div>
+      <p><i>Draft state</i> indicates the state of the deliverable at the time of the charter approval. <i>Expected
+          completion</i> indicates when the deliverable is projected to become a Recommendation, or otherwise reach a
+        stable state.</p>
 
-         <p style="padding: 0.5ex; border: 1px solid green"> This proposed charter is available
-      on <i class="todo"><a href="https://github.com/w3c/@@">GitHub</a>.
+      <section id="normative">
+        <h3>
+          Normative Specifications
+        </h3>
+        <p>
+          The Working Group will deliver the following W3C normative specifications:
+        </p>
+        <dl>
+
+          <dt id="webxr-device-api-1" class="spec"><a href="https://immersive-web.github.io/webxr/"
+              target="_blank">WebXR Device
+              API</a></dt>
+          <dd>
+            <p>We will continue to maintain the core WebXR Device API but we don’t intend to add new features or make
+              substantial
+              changes.</p>
+            <p>This specification defines support for accessing virtual reality (VR) and augmented reality (AR) devices,
+              including sensors and head-mounted displays, on the Web.</p>
+            <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/standards/types#ED"
+                target="_blank">Editor's Draft</a></p>
```

Comment:
I believe we should put the latest status "Working Draft" once we passed FPWD..
e.g. https://w3c.github.io/charter-drafts/pe-2020.html#deliverables

Reply:
Changed it to working draft. Thanks @himorin